src/styles/header.scss

```diff
@@ -36,6 +65,26 @@
     @media screen and (max-width: $layout-breakpoint-desktop-xs) {
       max-width: 575px;
     }
+
+    @media screen and (max-width: $layout-breakpoint-tablet-m) {
+      display: none;
```

Comment:
I'm really not sure if this is the correct behavior, because, as we know, adaptive design in Figma is still not fixed and is still buggy.

I could suppose that originally there was intention to have burger version of header nav on $layout-breakpoint-tablet-m.
<img width="466" alt="image" src="https://github.com/thirdmadman/fond-ravnovesie.ru_2023/assets/21088474/da9bfd94-5080-4c99-8052-e77332a4f948">

Similar problem occurs on width ~1370px

I will fix this another way.